include/d/actor/d_a_tag_ba1.h

```diff
@@ -2,27 +2,40 @@
 #define D_A_TAG_BA1_H
 
 #include "f_op/f_op_actor.h"
+#include "m_Do/m_Do_hostIO.h"
 
 class daTag_Ba1_c : public fopAc_ac_c {
 public:
-    void XyCheck_cB(int);
-    void XyEvent_cB(int);
-    void createInit();
+    BOOL XyCheck_cB(int i_itemBtn);
+    int XyEvent_cB(int);
+    bool createInit();
     BOOL _draw();
     BOOL _execute();
     BOOL _delete();
     cPhs_State _create();
 
 public:
-    /* Place member variables here */
-};
+    /* 0x290 */ s16 field_0x290;
```

Comment:
Given the way this gets accessed, this should probably be typed as an array of a single short: `s16[1]`